mlflow/pyfunc/__init__.py

```diff
@@ -1745,6 +1746,34 @@ def _prebuild_env_internal(local_model_path, archive_name, save_path):
         shutil.rmtree(env_root_dir, ignore_errors=True)
 
 
+def _download_prebuilt_env_if_needed(prebuilt_env_path):
```

Comment:
Should we permit users to define the local dir where the env file is written to (defaulted to a tmp root) so that they can reuse an environment that they've built over a period of time instead of having to re-fetch and re-download if the tmp dir gets cleaned?

Reply:
this is a good idea :) I will update it.